apps/pie-microsite/src/_11ty/shortcodes/mediaElementList.js

```diff
@@ -0,0 +1,21 @@
+module.exports = ({ data }) => {
+    const listItems = JSON.parse(data).items;
+
+    return `<div class="c-mediaElementList">
+    ${listItems.map(({
+        headingLevel = 3, headingText, imgSrc, imgSrcNarrow, text
+    }) => {
+        const hx = `h${headingLevel}`;
+        return `<div class="c-mediaElement">
+            <picture>
+                <source srcset="${imgSrc}" media="(min-width: 600px)">
```

Comment:
nitpick: can we pop `600px` in a variable? if we do this elsewhere then I'd prefer this approach everywhere